server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java

```diff
@@ -184,9 +184,8 @@ public static String switchRootTableVolume(String location) throws IOException {
    * for use it chooses a new tablet directory.
    */
   public static TabletFiles updateTabletVolumes(AccumuloServerContext context, ZooLock zooLock,
-      VolumeManager vm, KeyExtent extent, TabletFiles tabletFiles, boolean replicate)
-      throws IOException {
-    List<Pair<Path,Path>> replacements = ServerConstants.getVolumeReplacements();
+      VolumeManager vm, KeyExtent extent, TabletFiles tabletFiles, boolean replicate,
+      List<Pair<Path,Path>> replacements) throws IOException {
     log.trace("Using volume replacements: " + replacements);
```

Comment:
Should the log statement be using parameter substitution? 
`log.trace("Using volume replacements: {}" + replacements);`

Reply:
Yeah but I was avoiding making unrelated changes to keep the change to a minimum.